src/overmap_ui.cpp

```diff
@@ -859,59 +859,101 @@ tripoint display( const tripoint &orig, const draw_data_t &data = draw_data_t()
 
             const std::string old_note = overmap_buffer.note( curs );
             std::string new_note = old_note, tmp_note;
+            const int max_note_length = 45;
+            const bool has_debug_vision = g->u.has_trait( trait_id( "DEBUG_NIGHTVISION" ) );
+
+            std::vector<std::pair<nc_color, long>> map_around;
+            map_around.reserve( 9 );
+            for( int i = 1; i < 4; i++ ) {
+                for( int j = 1; j < 4; j++ ) {
+                    const int omy = curs.y + i - 2;
+                    const int omx = curs.x + j - 2;
```

Comment:
There's no need for manual traversal: `map::points_in_radius` will do this for you.

Reply:
It looks like `map::points_in_radius` works with different coordinates.
I'm getting one element when the cursor is far from player position.

Reply:
Fixed in dac51cc